cordova-common/src/superspawn.js

```diff
@@ -22,7 +22,7 @@ var fs = require('fs');
 var path = require('path');
 var _ = require('underscore');
 var Q = require('q');
-var shell = require('shelljs');
+var which = require('shelljs/src/which');
```

Comment:
It looks like this relies on shelljs internal layout that may change in a future version.